### PR TITLE
Replace java.util.Logger with logging facade

### PR DESCRIPTION
**Pull request by ddsharpe**
Adding copy of facade from main code line.  Logging should be the same(similar) in both integration tests as main line.

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java [hunk 1]
```diff
@@ -351,8 +349,7 @@ public static boolean deleteNamespace(String name) throws ApiException {
 
     if (response.getObject() != null) {
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java:350]: i know you didn't touch this, but this seems pretty unreliable - we often see issues with stuck namespaces -- we should strengthen this

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java [hunk 2]
```diff
@@ -0,0 +1,629 @@
...
+import java.util.Arrays;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/** Centralized logging for the operator. */
+public class LoggingFacade {
+
+  public static final String TRACE = "OWLS-KO-TRACE: ";
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java:16]: can we make this `WKO-TRACE`?

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java [hunk 3]
```diff
@@ -0,0 +1,629 @@
...
+  protected static final String CLASS = LoggingFacade.class.getName();
+  private final Logger logger;
+
+  /**
+   * Construct logging facade.
+   * @param logger logger
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java:22]: please improve javadoc

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java [hunk 4]
```diff
@@ -0,0 +1,629 @@
...
+
+  /**
+   * Converts an array to a loggable string.
+   *
+   * @param value the object to log
+   * @param password true if the value is a password that should not be logged
+   * @return a loggable string
+   */
+  public static Object convertArraysForLogging(Object value, boolean password) {
+    // Don't log passwords.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java:47]: sage advice

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java [hunk 5]
```diff
@@ -0,0 +1,629 @@
...
+    try {
+      return MessageFormat.format(logger.getResourceBundle().getString(msg), args);
+    } catch (Exception ex) {
+      return msg;
+    }
+  }
+
+  /**
+   * Obtains caller details, class name and method, to be provided to the actual Logger. This code
+   * is adapted from ODLLogRecord, which should yield consistency in reporting using PlatformLogger
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java:599]: this comment looks like it is incorrect

File: new-integration-tests/src/test/resources/OperatorIntegrationTests.properties [hunk 6]
```diff
@@ -0,0 +1 @@
+WLSIT-0000=WLS Operator Integration Tests
```
Comment (markxnelson) [inline new-integration-tests/src/test/resources/OperatorIntegrationTests.properties:1]: can we use `WKO-TEST-0000`??

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java (hunk 2 above)
Comment (ddsharpe) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java:16]: This is a copy of the LoggingFacade object in the main line.  You want to have a different value in this version of the facade than the one in mainline?  But, I do like WKO better than OWLS-KO

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java (hunk 3 above)
Comment (ddsharpe) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java:22]: Not my Javadoc...

File: new-integration-tests/src/test/resources/OperatorIntegrationTests.properties (hunk 6 above)
Comment (ddsharpe) [reply new-integration-tests/src/test/resources/OperatorIntegrationTests.properties:1]: Doubt we will ever use it at all, but sure.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java (hunk 5 above)
Comment (ddsharpe) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java:599]: You added it to the code.... You should know.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java (hunk 1 above)
Comment (markxnelson) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java:350]: can we create a JIRA to fix this please?

**Pull request merged**